sass/includes/_related-resources.scss

```diff
@@ -18,38 +18,53 @@
   }
 
   &__list-item {
-    background-color: $color__dark;
+    background-color: $color__light-grey;
     margin-bottom: 1rem;
-    padding: 1rem;
-    border-left: 1.25rem solid $color__yellow_2;
+    padding: 1rem 1.5rem;
+    border-left: 0.25rem solid $color__yellow_2;
     text-align: left;
+
+    @media (min-width: map-get($grid-breakpoints, "md")) {
+        border-left-width: 0.3125rem;
+    }
   }
 
   &__link {
-    text-decoration: none;
     transition: none;
+    text-decoration-thickness: 1px;
 
     &:hover {
-      text-decoration: underline;
+      text-decoration: none;
       color: $color__white;
     }
 
-    &:focus {
-      outline: 0.313rem solid $color__white;
-      outline-offset: .25rem;
-    }
+    // &:focus {
+    //   outline: 0.313rem solid $color__white;
+    //   outline-offset: .25rem;
+    // }
   }
 
   &__link-heading {
     margin-top: 0;
-    font-size: 1.625rem;
+    font-family: $font__body;
+    font-weight: $font-weight-bold;
+    font-size: 1.125rem;
+
+    @media (min-width: map-get($grid-breakpoints, "md")) {
+        font-size: 1.25rem;
+    }
   }
 
   &__link-text {
-    font-size: 1.2rem;
+    font-family: $font__body;
+    font-size: 1rem;
+
+    p {
+        margin-bottom: 0;
+    }
   }
 
   &__link, &__link-text {
-    color: $color__white !important;
+    color: $color__black !important;
```

Comment:
shame we need to use !important, but guessing there's no other easy way around this?

Reply:
I've removed this now :) wasn't needed!